fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTest.java

```diff
@@ -1190,18 +1189,20 @@ private enum SortType {
     static Stream<Arguments> continuationDuringRepartitioningTest() {
         return Stream.of(true, false)
                 .flatMap(grouped -> Stream.of(true, false)
-                        .flatMap(synthetic -> Arrays.stream(SortType.values())
-                                .map(sortType -> Arguments.of(grouped, synthetic, sortType))));
+                        .flatMap(uniqueTimestamps -> Stream.of(true, false)
+                                .flatMap(synthetic -> Arrays.stream(SortType.values())
+                                        .map(sortType -> Arguments.of(grouped, synthetic, uniqueTimestamps, sortType)))));
```

Comment:
We should remove the code in https://github.com/FoundationDB/fdb-record-layer/blob/86e1659b20609765ac3394eb49699788845b9ed8/fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java#L148-L151 that protects against duplicate timestamps.
This will probably require updating the sorting logic in https://github.com/FoundationDB/fdb-record-layer/blob/86e1659b20609765ac3394eb49699788845b9ed8/fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexTestValidator.java#L97 so that it expects the right thing when there are duplicate timestamps.